docs/developer-guide/implement-a-udf.rst

```diff
@@ -0,0 +1,349 @@
+.. _implement-a-udf:
+
+Implement a User-defined Function (UDF)
+#######################################
+
+Prerequisites
+     - `Apache Maven <https://maven.apache.org/download.cgi>`__
+     - |cp| installed locally

```

Comment:
make this CP download a hyperlink to https://docs.confluent.io/current/installation/installing_cp/index.html